src/main/java/uk/gov/companieshouse/company/profile/service/CompanyProfileService.java

```diff
@@ -111,15 +111,23 @@ public void updateInsolvencyLink(String contextId, String companyNumber,
             ApiResponse<Void> response = insolvencyApiService.invokeChsKafkaApi(
                     contextId, companyNumber);
 
-            if (response.getStatusCode() == HttpStatus.OK.value()) {
+            HttpStatus statusCode = HttpStatus.valueOf(response.getStatusCode());
+
+            if (statusCode.is2xxSuccessful()) {
                 logger.info(String.format("Chs-kafka-api CHANGED invoked successfully for "
                         + "contextId %s and company number %s", contextId, companyNumber));
                 companyProfileRepository.save(cpDocument);
+                logger.trace(String.format("Company profile is updated in MongoDB with "
+                        + "contextId %s and company number %s", contextId, companyNumber));
+            } else {
+                logger.info(String.format("Chs-kafka-api CHANGED NOT invoked successfully for "
+                        + "contextId %s and company number %s", contextId, companyNumber));
+                logger.trace(String.format("Company profile is NOT updated in MongoDB with"
+                        + " contextId %s and company number %s", contextId, companyNumber));
```

Comment:
We don't tend to emit negative logs as it should be implicit - no success log then something went wrong - usually an error with more details (and maybe an error log) will be thrown anyway - particularly for the chs-kafka-api call. Happy with the trace level one